core/templates/pages/exploration-editor-page/services/exploration-save.service.spec.ts

```diff
@@ -30,6 +30,7 @@ import { ChangeListService } from './change-list.service';
 import { ExplorationDataService } from './exploration-data.service';
 import { ExplorationLanguageCodeService } from './exploration-language-code.service';
 import { ExplorationTagsService } from './exploration-tags.service';
+import { NgbModal, NgbModalRef } from '@ng-bootstrap/ng-bootstrap';
```

Comment:
Move this import to be below the `@angular` import statements.

Reply:
Done